tbx/project_styleguide/templates/patterns/atoms/avatar/avatar.html

```diff
@@ -1,4 +1,4 @@
 {% load wagtailimages_tags %}
 <div class="avatar{% if classes %} {{ classes }}{% endif %}">
-    {% image avatar width-143 format-webp class="avatar__image" %}
+    {% image avatar width-143 format-webp loading="lazy" class="avatar__image" %}
```

Comment:
On a blog page we probably don't want to lazy load the avatar as it will be above the fold ([web.dev link](https://web.dev/articles/browser-level-image-lazy-loading#avoid_lazy_loading_images_that_are_in_the_first_visible_viewport)) - we could make adding the attribute optional for components that can lazy load it? Might have to output the `<img>` tag in instance.

Reply:
ah I will look at doing this on the avatar changes pr.